### ISSUE TITLE
Create schema in Mysql workbench

### ISSUE THREAD

**Issue by patroklossam** [patroklossam/WareHouse#50]

**Comment by anwaysomani** [comment]
I would like to perform the schema making task, and wish to know more about how to proceed, as I am entirely new to the OSS.

**Comment by patroklossam** [comment]
Hi! sure feel free.

you will need 2 tools for that, 
SQLite studio to open the mysqlite database in the db folder and check the tables and columns that exist,
and Mysql workbench to create the model. you will save the file and place it in the db folder of the repository. 

Are you familiar with git? can you follow the procedure described in the readme file?

Feel free to ask anything you need!
Patroklos

**Comment by patroklossam** [comment]
Just remember to create the commits and pull request after the 1st of October to count in your contributions for Hacktoberfest!

**Comment by anwaysomani** [comment]
Yes, I am familiar with the git environment, and I shall surely make my commits after 1 October. Till then, I shall prepare the required resources to commit.

**Comment by patroklossam** [comment]
awesome! happy hacktoberfest then! I am looking forward to your contribution!
